test/services/nexboard/index.test..js

```diff
@@ -0,0 +1,220 @@
+const chai = require('chai');
```

Comment:
there are a bit too many dots in the filename xD

Reply:
xD